RELEASE/scripts/autoscend/combat/auto_combat_default_stage3.ash

```diff
@@ -250,30 +250,127 @@ string auto_combatDefaultStage3(int round, monster enemy, string text)
 
 		if(my_location() == $location[The Smut Orc Logging Camp] && canSurvive(1.0) && get_property("chasmBridgeProgress").to_int() < 30)
 		{
+			boolean coldMortarShell = canUse($skill[Stuffed Mortar Shell]) && have_effect($effect[Spirit of Peppermint]) != 0;
+			skill coldSkillToUse;
 			// Listed from Most to Least Damaging to hopefully cause Death on the turn when the Shell hits.
-			if(canUse($skill[Stuffed Mortar Shell]) && have_effect($effect[Spirit of Peppermint]) != 0)
+			if(canUse($skill[Saucegeyser], false))
```

Comment:
Per wiki, Saucegeyser randomly does hot or cold damage. Exception is if one element does more damage. Wiki also says only the pervert is sleaze aligned (extra cold damage). All the other orcs don't have an element. So I'd expect hot and cold damage to be randomly picked for the non-pervert orcs.

Wiki could be wrong - did you confirm Saucegeyser consistently does cold damage?

Reply:
> Wiki could be wrong - did you confirm Saucegeyser consistently does cold damage?

it does not. though I didn't add Saucegeyser, just the MP cost check? strangely autoscend isn't relying on Icy Glare to make it cold
going to make a small change to spell priority for this